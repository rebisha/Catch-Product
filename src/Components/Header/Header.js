import React from "react";

import styles from "./header.scss";

const Header = () => {
  return (
    <header className={styles.header}>
      <a href="https://www.catch.com.au/" target="_blank" rel="noreferrer">
        <img
          className={styles.header__image}
          src="https://s.catch.com.au/static/catch/images/logo-83d9b21199.svg"
          alt="logo"
        />
      </a>
    </header>
  );
};

export default Header;
