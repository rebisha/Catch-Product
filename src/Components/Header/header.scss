@import "../../assets/styles/variables";
@import "../../assets/styles/functions";

.header {
  padding: units(20px 64px);

  &__image {
    max-width: 200px;
    height: auto;
    object-fit: contain;
  }
}
