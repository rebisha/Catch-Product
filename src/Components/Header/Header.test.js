import React from "react";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";
// component
import Header from "./Header";

it("Should have an image", () => {
  const wrapper = shallow(<Header />);
  expect(wrapper.find("img").exists()).toBe(true);
});

it("Should render correctly", () => {
  const wrapper = renderer.create(<Header />);
  expect(wrapper).toMatchSnapshot();
});
