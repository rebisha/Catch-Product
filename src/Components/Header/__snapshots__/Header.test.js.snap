// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correctly 1`] = `
<header
  className="header"
>
  <a
    href="https://www.catch.com.au/"
    rel="noreferrer"
    target="_blank"
  >
    <img
      alt="logo"
      className="header__image"
      src="https://s.catch.com.au/static/catch/images/logo-83d9b21199.svg"
    />
  </a>
</header>
`;
